content/yaml-running-builds/starting-builds-automatically.md

```diff
@@ -205,6 +205,32 @@ workflows:
       condition: not event.pull_request.draft
 {{< /highlight >}}
 
+Example 2. This build will *not* continue if the pull request branch has a "codemagicTest" label:
+
+{{< highlight yaml "style=paraiso-dark">}}
+workflows:
+  build:
+    name: Build on PR update
+    triggering:
+      events:
+        - pull_request
+    when:
+      condition: not event.pull_request.labels[0].name == "codemagicTest"
+{{< /highlight >}}
+
+Example 3. You can also combine triggering events, just make sure that each condition is wraped with brackets:
```

Comment:
Example 3. You can also combine triggering conditions, just make sure that each condition is wrapped with brackets: